std_msgs/Bool trigger
---
sensor_msgs/PointCloud2 input_cloud
